widgets/text/text.go

```diff
@@ -108,6 +109,25 @@ func (t *Text) Write(text string, wOpts ...WriteOption) error {
 	if opts.replace {
 		t.reset()
 	}
+
+	// If set, limit the buffer (if maxContent has been set)
+	if len(t.content) >= t.opts.maxContent && t.opts.maxContent > 0 {
```

Comment:
What do we do when someone sets the option to zero cells? Should that be an error when the option is being set?

Can we also add a test case?

Reply:
Easiest fix is probably to set the conditional logic to >= 0. It'll then be ignored since the only valid size is 1 or greater.